tests/apps/core/test_authentication_delegation.py

```diff
@@ -28,8 +28,7 @@ class UserMenuTests(CMSTestCase):
     )
     def test_user_menu_with_profile_urls(self):
         """
-        user menu should render correctly
-        and includes formatted profile_urls
+        User menu should render correctly and includes formatted profile_urls
```

Comment:
Add the explanation so that we are not surprised by the way you test these strings :wink: 
```suggestion
        User menu should render correctly and includes formatted profile_urls.
        Characters should be escaped with the `escapejs` template tag to prevent
        an issue when this variable contains unescaped characters for javascript (e.g: '\').
```